docs/csharp/tutorials/exploration/records.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Use record types - C# tutorial
+description: This tutorial teaches you how to use record types, build hierarchies of records, and when to choose records over classes.
+ms.date: 11/12/2020
+---
+# Create record types
+
+C# 9 introduces *records*, a new reference type that you can create instead of classes or structs. Records are distinct from classes in the record types use *value-based equality*. Two variables of a record type are equal if the record types are identical, and if for every field, the values in both records are equal. Two variables of a class type are equal if the objects referred to are the same class type and the variables refer to the same object. That design decision implies other capabilities you'll probably want in record types. The compiler generates many of those members when you declare a `record` instead of a `class`.
```

Comment:
```suggestion
C# 9 introduces *records*, a new reference type that you can create instead of classes or structs. Records are distinct from classes in that record types use *value-based equality*. Two variables of a record type are equal if the record type definitions are identical, and if for every field, the values in both records are equal. Two variables of a class type are equal if the objects referred to are the same class type and the variables refer to the same object. Value-based equality implies other capabilities you'll probably want in record types. The compiler generates many of those members when you declare a `record` instead of a `class`.
```

Not sure what "those members" refers to in the last sentence. Members for the capabilities?